comboomPunkTsucht/comboom.sucht-BLOG
# Add blog page, RSS feed, and various fixes for syncing fork and project removal

This PR includes the following changes:

    Upgrade next.js from 14.3.0-canary.87 to 15.0.0-rc.0.
    Introduced experimental react compiler by adding babel-plugin-react-compiler.
    Fixed issues with generateStaticParams by removing it from next.config.mjs.
    Enhanced data fetching in blog page to ensure proper async operation.
    Improved error handling in getSortedPostsData by skipping directories.
    Updated various dependencies in package.json and yarn.lock for better compatibility.
    Corrected minor typos and formatting in markdown files.

### How to test?
1. Run the application using the updated next.js version.
2. Navigate through various blog pages to ensure proper rendering.
3. Verify RSS feed generation with the updated data.
4. Confirm all dependencies are correctly installed and application runs seamlessly.

---
